### ISSUE TITLE
[imagick] Strip GPS data from EXIF if present

### ISSUE THREAD

**Issue by imorland** [FriendsOfFlarum/upload#304]
With https://github.com/flarum/core/pull/3195, it will be possible to use `imagick` as the driver for `Intervention/image`, which is used to resize, orientate, etc images.

As pointed out https://github.com/flarum/core/pull/3195#issuecomment-991967527, this could be an issue moving forward.

Let's at least add an option to strip GPS data from uploaded images.

**Comment by clarkwinkelmann** [comment]
Makes sense. Should the option directly be to erase all exif data? That way every potential info is covered with a single setting.
